api/osrd_infra/migrations/__init__.py

```diff
@@ -2,42 +2,84 @@
 from django.db import migrations, models
 
 
-def run_sql_complexe_add_foreign_key(table_name: str, field_name: str, model_name: str, link_model: str, link_table: str, nullable: bool = False):
+def run_sql_complexe_add_foreign_key(
+    table_name: str,
+    field_name: str,
+    model_name: str,
+    link_model: str,
+    link_table: str,
+    nullable: bool = False,
+    related_name: str = None,
+):
+
+    related_name = {"related_name": related_name}
+    field = models.ForeignKey(
+        on_delete=django.db.models.deletion.CASCADE,
+        to=f"osrd_infra.{link_model}",
+        null=nullable,
+        blank=nullable,
+        **related_name,
+    )
```

Comment:
I didn't know that related_name was `None` by default. Since it the case you can do the following:

```suggestion
    field = models.ForeignKey(
        on_delete=django.db.models.deletion.CASCADE,
        to=f"osrd_infra.{link_model}",
        null=nullable,
        blank=nullable,
        related_name=related_name,
    )
```